plugin/completion.py

```diff
@@ -125,3 +99,26 @@ def translated_regions(self, edit_region: sublime.Region) -> Generator[sublime.R
             translation = region.b - primary_cursor_position
             translated_edit_region = sublime.Region(edit_region.a + translation, edit_region.b + translation)
             yield translated_edit_region
+
+    def epilogue(self, item: CompletionItem, session_name: str) -> None:
+        session = self.session_by_name(session_name, 'completionProvider.resolveProvider')
+        if session:
+            request = Request.resolveCompletionItem(item, self.view)
+            session.send_request_async(request, lambda response: self.on_resolved(response, session_name))
```

Comment:
I think it would be good to jump back on the main thread before calling `on_resolved`.

Reply:
Can you give me an answer of what will that change(moving to the main thread) improve(or fix)? (I am bad at threads)

Reply:
`on_resolved` is expected to run on the main thread so it's more about the expectations than an actual issue.

Even though I don't remember the specific case, I remember that there were issues running LSP APIs or commands on async thread. Especially the ones that do some view edits.

Reply:
Thanks, done with 78fe8bb